.eslintrc

```diff
@@ -2,12 +2,20 @@
     "env": {
         "node": true
     },
+    "extends": "eslint:recommended",
     "rules": {
-        "quotes": [1, "single"],
-        "strict": 0,
-        "no-mixed-requires": 0,
-        "no-underscore-dangle": 0,
-        "no-shadow": 0,
-        "new-cap": 0
+        "no-mixed-requires": [2, { "allowCall": true }],
+        "quotes": [2, "single"],
+        "no-trailing-spaces": 2,
+        "indent": 2,
+        "linebreak-style": [2, "unix"],
+        "semi": [2, "always"],
+        "brace-style": [2, "1tbs", { "allowSingleLine": true }],
+        "keyword-spacing": 2,
+        "space-before-blocks": 2,
+        "space-before-function-paren": [2, { "anonymous": "always", "named": "never" }],
+        "no-mixed-spaces-and-tabs": 2,
+        "comma-spacing": [2, {"before": false, "after": true}],
+        "key-spacing": [2, {"beforeColon": false, "afterColon": true}]
     }
 }
```

Comment:
Thought about using eslint-config-homeoffice?
```
extends:
- "homeoffice/config/default"
env:
  es6: true
```
certainly cuts down a lot of "stuff"